README.rst

```diff
@@ -331,6 +331,18 @@ Another financial institution with short password requirements. They also block
 
 |Commsec|
 
+`Copart <https://copart.com>`__
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Copart: "The security of our members is extremely imporant to us."
+
+Also Copart: "We're gonna need you to keep your password between 5-10 characters."
+
+|Copart1|
+
+They have an undocumented password history rule too.
+
+|Copart2|
```

Comment:
I think just the first screenshot is fine here as I wouldn't consider the password history as truly dumb.

Reply:
Thanks for the feedback! I've removed the second screenshot.